src/dmd/semantic3.d

```diff
@@ -1378,4 +1347,46 @@ private struct FuncDeclSem3
             }
         }
     }
+    
+    /* Create scope to avoid name collisions
+     */
+    Scope* setupScope()
```

Comment:
`private`

Reply:
Its part of a private struct already, does it really need it?

Reply:
Ah didn't see that due to the diff, sorry. I guess it doesn't need it, but it doesn't hurt readability (I see you already added it).

Reply:
Only 'cos I got hit by the stupid whitespace checker.